src/ui/js/Overview.js

```diff
@@ -213,25 +211,37 @@ Overview.pageAddGameTable = function(
     sectionHeader,
     reverseSortOrder
   ) {
-  if (gameType == 'new') {
-    Overview.pageAddGameTableNew();
-    return;
-  }
-
-  if (typeof reverseSortOrder === 'undefined') {
-    reverseSortOrder = false;
-  }
-
   var gamesource;
   var tableClass;
-  if (gameType == 'finished') {
-    gamesource = Api.completed_games.games;
-    tableClass = 'finishedGames';
-  } else if (gameType == 'rejected') {
-    gamesource = Api.rejected_games.games;
-    tableClass = 'rejectedGames';
-  } else {
-    gamesource = Api.active_games.games[gameType];
+
+  switch(gameType) {
+  case 'closed':
+    // closed games comprise completed games and rejected games
+    var gamesourceCompleted = Api.completed_games.games;
+    Overview.addTypeToGameSource(gamesourceCompleted, 'completed');
```

Comment:
Hmm... do we want to do this in Overview.js, or should `Api.packageGameData()` take care of it instead?  Probably either way is reasonable, it just feels a little hackish/verbose to do it here, and might be more natural in `packageGameData`